test/app/view/web/WebChessBoardView.spec.ts

```diff
@@ -73,8 +73,8 @@ describe('Web Chess Board View with starting pieces positions', () => {
     const image = g8WBlackKnight.getAttribute('src');
 
     expect(g8Square).toContainElement(g8WBlackKnight);
-    expect(image).not.toBe('static/img/pieces/black-pawn.svg');
-    expect(image).toBe('static/img/pieces/black-knight.svg');
+    expect(image).not.toBe('../static/img/pieces/black-pawn.svg');
+    expect(image).toBe('../static/img/pieces/black-knight.svg');
```

Comment:
Czy tego dotyczy PR?

Reply:
W sumie na developie nam działa, to skad sie bierze ta zmiana?

Reply:
możliwe, że źle tutaj coś zrobiłem przy konfliktach .... @tomdworniczak tutaj powinno być 
`    expect(image).not.toBe('static/img/pieces/black-pawn.svg');
    expect(image).toBe('static/img/pieces/black-knight.svg');`

Reply:
mam to pozmieniać ? we wszystkich testach?

Reply:
hmmm w sumie w testach to chyba ścieżki nie trzeba zmieniać... W głównych plikach tak, żeby działały obrazki po deploy'u (wgl muszę to sprawdzić, bo zapomniałem wczoraj), ale w testach też zmianiamy @nowakprojects?

Reply:
Nie wiem, zrób tak, żeby działało :)